rules/concepts.md

```diff
@@ -1,25 +1,50 @@
-# Cover generic rules concepts
+# Rules Concepts
 
-Pull stuff from the Getting Started Tutorial.
+openHAB is about home automation, but to create home automation we need to define rules.
+
+{::options toc_levels="2..4"/}
+
+- TOC
+{:toc}
 
 ## What Are Rules
 
+You can think of rules as routines or behaviours for your smart home.
+
+Many people have a routine when they wake up in the morning: make the bed, make coffee, make and eat breakfast, brush their teeth, etc.
+Similarly your smart home can have a routine: when the sun rises raise the blinds and adjust the temperature.
+Another routine could be to turn off all the lights and turn down the temperature when you leave home:
+openHAB can do that for you.
+There is no need to open your app and switch of the lights manually.
+Do you want to hear your favorite music when you arrive back at home?
+No problem, openHAB can do that for you too.
+Rules can also remind you of things, for example, that you opened the window hours ago and forgot that it is open?
+openHAB can send a notification to your phone.
 ## Parts of a Rule
 
-### Triggers
+These rules take the high level form of *When __t__ happens, if __c__ then do __a__*,
+where __t__ are the _triggers_ that cause the rule run,
+__c__ are the _conditions_ that must be true for the rule to run, and __a__ are _actions_ to perform when the rule is triggered and allowed to run.
+Note that both __t__ and __c__ can be optional.
 
-### Conditions
+To work with the *When __t__ happens, if __c__ then do __a__* principle, openHAB rules consist of three parts:
 
-### Script Actions
+| Letter | Name        | Purpose                                                                                |
+|--------|-------------|----------------------------------------------------------------------------------------|
+| __t__  | `Trigger`   | The *When __t__ happens* part: It causes the rule run when the defined event happens.  |
```

Comment:
Let's add a column between Name and Purpose called "Rule Part" and move the "When __t__ happens" stuff to that column. Visually, I think that separation will help with comprehension.

| Letter | Name | Rule Part | Purpose|
|-|-|-|-|
| __t__ | `Trigger` | *When __t__ happens* | Causes the rule to run when the defined event happens. |

Or maybe it would be even better to replace the "Letter" column with the Rule Part.